File: benchmark/perl/geometric_series.pl
use Time::HiRes qw(time);
my ($r, $sum, $term) = (0.999, 0, 1);
my $start = time();
for (1..1000000) { $sum += $term; $term *= $r; }
my $end = time();
my $ms = ($end - $start) * 1000;
print "Sum: $sum
";
printf "Time: %.0fms
", $ms;

File: benchmark/lua/geometric_series.lua
local start = os.clock()
local r = 0.999
local s = 0.0
local term = 1.0
for _ = 1, 1000000 do
    s = s + term
    term = term * r
end
local finish = os.clock()
print("Sum: " .. s)
print(string.format("Time: %.0fms", (finish - start) * 1000))

